_data/projects/linkerd.yml

```
@@ -0,0 +1,19 @@
+name: Linkerd
+desc: An ultralight service mesh for Kubernetes.
+site: https://linkerd.io/
+tags:
+- oss
+- web
+- cloud-native
+- golang
+- go
+- javascript
+- kubernetes
+- rust
+- CNCF
+- containers
+- docker
+- react
+upforgrabs:
+  name: good first issue
+  link: https://github.com/linkerd/linkerd2/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

:art: this is a cleaner URL that will give you the same result:

```suggestion
  link: https://github.com/linkerd/linkerd2/labels/good%20first%20issue
```

---

Cool! I didn't realize that only selected open issues =)